### PR TITLE
Refactor backend challenge

### PR DESCRIPTION
**Pull request by Stuwert**
- Removes all files related to code and database concerns. We don't use this anymore, so it's unnecessary.
- Renames teams -> businesses and transactions -> employees.  Trying to make nomenclature a little bit easier for interviewees.  The teams to businesses transition seemed a bit less necessary to me, but my bias was towards making the brief understandable.
- Reframed the problem in terms of a business issue, rather than starting with a data structure.
- Restructured the extra credit to be further down the list of concerns and made the different sections explicit.
- Created an example get request.
- Explicitly included "writing a sql query" in the list of core requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,104 @@
...
-- Modify your custom field schema to support multi-select multiple choice. (For example, a user should be able to specify multiple _Cost Centers_ to a single transaction.)
-- Add an endpoint for updating a transaction with new custom data.
-- Consider the performance of your solution when the transactions database begins to contain millions of rows. Ensure that your solution scales elegantly to prevent increased latency on the transactions endpoint with time.
-- There's no security, so anyone can use your new app! For example, one team can currently edit or delete another team's data (BAD!). Try to implement some sort of authentication or authorization.
-- The `GET /transactions` request is going to get quite large as teams continue to use this project. Add a paremeter to this request that allows the user to request specific date ranges, as well as request the report year-to-date, month-to-date, and week-to-date.
+The goal of this task is to build an understanding of how you would structure databases and APIs, so no coding is actually necessary. We will use a whimsical doc and `NOTES.md` to write out your thoughts.
```
Comment (sbassin) [inline README.md]: _Can_ we share a Whimsical doc with the candidate and have them edit it?

File: README.md [hunk 2]
```diff
@@ -1,50 +1,104 @@
...
-#### Core Requirements
+Dunder Mifflin wants to store custom fields _Sales Ranking_ and _Michael Scott Rating_. 
 
-- Implement a method to store custom data on a transaction.
-  - Each team needs to be able to assign a custom name to each of their fields, as well as indicate how many custom fields they would like to use.
-  - Custom fields should have values that are either: (a) plain text or (b) single-select multiple choice. Teams should be able to indicate what type of value each of their fields should accept. (It is out of scope of this project to design a front-end UI for the selection of these options)
-- Add endpoints for the creation, editing, viewing, and deletion of a team's custom field schema
-- Enable users to specify their custom field values on a `POST /transactions` request.
-- Return the associated values of the custom fields on the `GET /transactions` request.
+Other businesses may want more custom fields, other companies may want fewer.  There may be between 0 to N custom fields per business.
```
Comment (sbassin) [inline README.md]: Would it make sense to add this?
```suggestion
Other businesses may want more custom fields, other companies may want fewer.  There may be between 0 to N custom fields per business. The solution you choose should support these other businesses' custom fields without requiring additional engineering work.
```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,104 @@
...
 
-### Installation
+### Before You Get Started
 
-- `$ createdb pana_backend_challenge`
-- `$ cp .env.sample .env`
-- `$ npm install`
-- `$ npm run db:init`
-- `$ npm start`
+We recognize that doing anything within a tight time constraint is an extra level of challenge that you won't face in your day to day job as an engineer.  The goal of this challenge isn't for you to complete everything but for us to get a sense of how you solve problems, so if you don't end up finishing the "core requirements" in the time allotted that is not immediate cause for concern.  Please ask any clarifying questions and work through your thought process as visibly as possible.  We're excited to see what you create!
```
Comment (sbassin) [inline README.md:23]: I like calling out the fact that the interview may be successful without finishing everything as well as the request for asking clarifying questions and thinking out loud. 👍🏼

File: README.md [hunk 4]
```diff
@@ -1,50 +1,104 @@
...
 
-### Submission
+## Requirements
 
-When you're finished, zip the directory (excluding the node_modules) and email it to devon@pana.com. Please do not create a public repository with your solution, as we'd like to discourage others from using your code. 😁
+### Core Requirements
 
-With your submission, please include the approximate number of hours you spent working on this. Additionally, if you have any feedback for us about how the process went, we'd love to hear it! We're always looking to imrpove.
+- Implement a method to store custom data related to an employee.
```
Comment (sbassin) [inline README.md]: Are we actually asking for a schema design here? I see the question below about writing a query but nothing about the table structure.

```suggestion
- Design a schema to store custom data related to an employee.
```

**Review by sbassin — APPROVED** [review]
I added some minor questions and suggestions. Overall this seems like a nice improvement.

File: README.md (hunk 1 above)
Comment (Stuwert) [reply README.md]: Oh yeah I need to remove this line. We cannot :(

**Review by bryan-pana — APPROVED** [review]

**Pull request merged**